network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -52,7 +52,7 @@ class InitiativeSection(models.Model):
         max_length=250,
     )
 
-    sectionButtonURL = models.TextField(
+    sectionButtonURL = models.URLField(
```

Comment:
Let's revert this, because URLField only works for fully qualified URLs, not relative URLs.

Reply:
Ahh yes, solid point. So should we revert _all_ of these back to TextFields? And is there a reason we can't use a CharField?

Reply:
In models we can easily add the field to `override_translatable_fields`, but in customblocks, this was the only way to exclude those fields. I guess we will need https://github.com/wagtail/wagtail-localize/issues/307 to be fixed first for these

Reply:
@KalobTaulien let's not refactor any of the field types in this PR, and only focus on the non-extraction part.

Reply:
Roger that.